src/Utils/request/request.ts

```diff
@@ -0,0 +1,27 @@
+import { RequestOptions, Route } from "./types";
+import { makeHeaders, makeUrl } from "./utils";
+
+interface Options extends RequestOptions {
+  controller?: AbortController;
+}
+
+export default async function request<TData>(
```

Comment:
We already have `fireRequest` to making http request to care endpoints, do we have a need for another request function that uses fetch? 😅

Reply:
I understand, another question, although we don't need axios for this case, but since we already use it for firerequest should we use it here too for consistency? (btw, I remember we planned to switch from axios to fetch but not sure exactly what happened to that PR). Anyhow, LGTM.

Reply:
we still can continue #6003 and get Axios removed. cc: @sainak